versioned_docs/version-2.0/self_hosting/configuration/blob_storage.mdx

```diff
@@ -0,0 +1,98 @@
+import {
+  CodeTabs,
+  HelmBlock,
+} from "../../../../src/components/InstructionsWithCode";
+
+# Enable blob storage
+
+By default, LangSmith stores run inputs, outputs, and errors in ClickHouse. In addition, LangSmith will store run manifests in Postgres. In high trace environments, this may balloon the size of your databases.
+To alleviate this, LangSmith supports storing run manifests, inputs, outputs, and errors in an external blob storage system.
```

Comment:
Add also on data privacy?  (storing it in your own bucket)

Reply:
added